storage/database/database_test.go

```diff
@@ -105,7 +105,7 @@ func TestDatabase_NotFoundErr(t *testing.T) {
 		val, err := db.Get(randStrBytes(100))
 		assert.Nil(t, val)
 		assert.Error(t, err)
-		assert.Equal(t, dataNotFoundErr, err)
+		assert.Equal(t, err, dataNotFoundErr)
```

Comment:
Is this intended? Isn't it `assert.Equal(t, expected, actual)`?

Reply:
Oh, you're right. Let me fix this. Thanks!

Reply:
Thanks :)